scene/3d/audio_stream_player_3d.cpp

```diff
@@ -831,7 +832,7 @@ float AudioStreamPlayer3D::get_attenuation_filter_db() const {
 }
 
 void AudioStreamPlayer3D::set_attenuation_model(AttenuationModel p_model) {
-	ERR_FAIL_INDEX(p_model, 3);
+	ERR_FAIL_INDEX(p_model, 4);
```

Comment:
When I set the value to three, in the editor, None appears but I cannot select it. When the value is set to 4, then the build fails (gives warning).

Also it says value is always false, but shouldn't that be the case?